pkg/i18n/french.go

```diff
@@ -76,30 +76,30 @@ func frenchSet() TranslationSet {
 		StandaloneContainersTitle: "Conteneurs Autonomes",
 		ImagesTitle:               "Images",
 		VolumesTitle:              "Volumes",
-		CustomCommandTitle:        "Commande personalisé:",
-		BulkCommandTitle:          "Commande groupée:",
+		CustomCommandTitle:        "Commande personalisée :",
+		BulkCommandTitle:          "Commande groupée :",
 		ErrorTitle:                "Erreur",
 		LogsTitle:                 "Enregistrements",
```

Comment:
```suggestion
		LogsTitle:                 "Journaux",
```
On traduit généralement "system logs" par "journaux système", par exemple.